ranger-discovery-bundle/src/test/java/io/appform/ranger/discovery/bundle/id/IdGeneratorTest.java

```diff
@@ -181,7 +181,7 @@ void testConstraintFailure() {
 
     @Test
     void testNodeId() {
-        val generatedId = IdGenerator.generate("TEST123");
+        val generatedId = IdGenerator.generate("TEST");
```

Comment:
Why is this change required ?

Reply:
With this change, Parsers won't support parsing IDs with numeric prefix.

Reply:
Then current test should be mutated to reflect new assertions rather than changing the test input. Add more tests if required